### PR TITLE
test: improve test case

### PR DESCRIPTION
**Pull request by ClarkXia**
- [x] test: improve test case
- [x] fix: optimize code of runClientApp #508 
- [x] fix: remove useless resolve of css namespace
- [x] chore: fix type error

### CODE REVIEW

File: examples/rax-project/ice.config.mts [hunk 1]
```diff
@@ -3,5 +3,5 @@ import compatRax from '@ice/plugin-rax-compat';
 
 export default defineConfig({
   publicPath: '/',
-  plugins: [compatRax()],
+  plugins: [compatRax({ inlineStyle: true })],
```
Comment (wssgcg1213) [inline examples/rax-project/ice.config.mts:6]: 111

File: packages/runtime/tests/appConfig.test.ts [hunk 2]
```diff
@@ -1,13 +1,33 @@
...
+    expect(appConfig).toStrictEqual({
+      app: {
+        rootId: 'app',
+        strict: false,
+      },
+      router: {
+        type: 'browser',
+      },
+    });
+  })
```
Comment (wssgcg1213) [inline packages/runtime/tests/appConfig.test.ts:23]: 少了分号

File: packages/runtime/tests/appConfig.test.ts [hunk 3]
```diff
@@ -1,13 +1,33 @@
...
-  test('defineAppConfig', () => {
+  it('getAppConfig', () => {
+    const appConfig = getAppConfig({
+      default: {
+        app: {
+          rootId: 'app',
+        }
+      },
+      auth: {},
+    })
```
Comment (wssgcg1213) [inline packages/runtime/tests/appConfig.test.ts:13]: `;`

File: packages/runtime/tests/navigator.test.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  const staticNavigator = createStaticNavigator();
+  it('createHref', () => {
+    expect(staticNavigator.createHref('/')).toBe('/');
+  });
+
+  it('push', () => {
+    try {
+      staticNavigator.push('/');
+    } catch(err) {
+      expect(true).toBe(true);
```
Comment (wssgcg1213) [inline packages/runtime/tests/navigator.test.ts:14]: https://vitest.dev/api/#tothrowerror

File: examples/rax-project/ice.config.mts (hunk 1 above)
Comment (ClarkXia) [reply examples/rax-project/ice.config.mts]: fixed

File: packages/runtime/tests/navigator.test.ts (hunk 4 above)
Comment (ClarkXia) [reply packages/runtime/tests/navigator.test.ts]: fixed

**Comment by ClarkXia** [comment]
测试用例已全局 lint

**Review by wssgcg1213 — DISMISSED** [review]

**Review by wssgcg1213 — APPROVED** [review]

**Review by luhc228 — APPROVED** [review]

**Pull request merged**